src/main/java/net/sf/jabref/gui/fieldeditors/EntryLinkListEditor.java

```diff
@@ -471,7 +471,7 @@ public Object getValueAt(int rowIndex, int columnIndex) {
                 case 1:
                     return entry.getLinkedEntry()
                             .map(bibEntry -> formatEntry(bibEntry, entry.getDataBase()))
-                            .orElse("Unknown entry");
+                            .orElse(Localization.lang("Unknown entry"));
```

Comment:
No idea. But do not touch them in this PR. 
Only update **your Properties file**, as otherwise you break all other languages, too°!